Resources/public/js/views/services/ez-sectionserversideviewservice.js

```diff
@@ -48,7 +50,81 @@ YUI.add('ez-sectionserversideviewservice', function (Y) {
                     struct.content.get('id'), update, tasks.add()
                 );
             });
-            tasks.done(Y.bind(data.afterUpdateCallback, this));
+            
+            tasks.done(
+                Y.bind(this._assignSectionCallback, this, data.sectionId, e.selection, Y.bind(data.afterUpdateCallback, this))
+            );
+        },
+        
+        /**
+         * Assign section callback trigerring notification and calling given callback
+         * 
+         * @method _assignSectionCallback
+         * @protected
+         * @param {String} sectionId the section id
+         * @param {Array} contents the array of contents to which section is being assgined to
+         * @param {Function} callback the callback to call when other tasks are done
+         */
+        _assignSectionCallback: function (sectionId, contents, callback) {
+            this._assignSectionNotificationDone(sectionId, contents);
+            callback();
+        },
+        
+        /**
+         * Notification changed to *started* before assigning section to contents
+         * 
+         * @method _assignSectionNotificationStarted
+         * @protected
+         * @param {String} sectionId the section id
+         * @param {Array} contents the array of contents to which section is being assgined to
+         */
+        _assignSectionNotificationStarted: function (sectionId, contents) {
+            var notificationIdentifier = this._getAssignSectionNotificationIdentifier('assign-section', sectionId, contents);
+            this.fire('notify', {
+                notification: {
+                    identifier: notificationIdentifier,
+                    text: 'Assigning section to contents',
+                    state: 'started',
+                },
+            });
+        },
+        
+        /**
+         * Notification changed to *done* after assigning section to contents
+         * 
+         * @method _assignSectionNotificationDone
+         * @protected
+         * @param {String} sectionId the section id
+         * @param {Array} contents the array of contents to which section is being assgined to
+         */
+        _assignSectionNotificationDone: function (sectionId, contents) {
+            var notificationIdentifier = this._getAssignSectionNotificationIdentifier('assign-section', sectionId, contents);
+            this.fire('notify', {
```

Comment:
nitpick CS: please add an empty line between the variable declarations and the actual code